hazelcast/src/test/java/com/hazelcast/internal/util/phonehome/PhoneHomeIntegrationTest.java

```diff
@@ -94,11 +95,12 @@ public void initialise()
         when(dockerPath.toRealPath()).thenReturn(Paths.get(System.getProperty("user.dir")));
         when(kubernetesTokenPath.toRealPath()).thenReturn(Paths.get(System.getProperty("user.dir")));
 
-        cloudInfoCollector = new CloudInfoCollector("http://localhost:8080/latest/meta-data",
-                "http://localhost:8080/metadata/instance/compute?api-version=2018-02-01",
-                "http://localhost:8080/metadata.google.internal", kubernetesTokenPath, dockerPath);
+        int port = wireMockRule.port();
+        cloudInfoCollector = new CloudInfoCollector("http://localhost:" + port + "/latest/meta-data",
```

Comment:
nit: the "http://localhost:" + port + "/" part is repeated many times, may be extracted to variable "baseUrl"